packages/clerk-js/src/ui/components/ImpersonationFab/ImpersonationFab.tsx

```diff
@@ -90,7 +90,7 @@ const FabContent = ({ title, signOutText }: FabContentProps) => {
         localizationKey={signOutText}
         onClick={async () => {
           // clerk-js has been loaded at this point so we can safely access session
-          await signOut({ sessionId: session!.id });
+          await signOut({ sessionId: session.id });
```

Comment:
Is this change intentional?

Reply:
I think linter removes this when you make changes to the file, and then the tests break 😢

Reply:
It's the linter yes. I reverted.